homeassistant/components/home_connect/coordinator.py

```diff
@@ -143,7 +143,7 @@ async def _event_listener(self) -> None:
                                         settings[setting_key].value = event.value
                                     else:
                                         settings[setting_key] = GetSetting(
-                                            setting_key, event.value
+                                            setting_key, setting_key.value, event.value
```

Comment:
I think we should use keyword arguments for the models now so it's easier to read and understand the interface.

Reply:
Side note: Maybe we should even require that in the models so it'd easier to add new optional parameters without a breaking change?